md/deploy-bundle.md

```diff
@@ -40,6 +117,14 @@ If you are installing the Performance Subscription edition,
 you need to edit [`bonita-platform-init-community-custom.properties`](BonitaBPM_platform_setup.md)
 and change the value of the `activeProfiles` key to `'community,performance'`. No change is needed for the Community, Teamwork, or Efficiency edition.
 
-## Next steps
+## Database initialization
+We assume here that the database has already been [created and configured for Bonita BPM](database-configuration.md#database_creation).
+Once created and configured you need to initialize it using the setup tool provided in the deploy bundle archive.
+This will create database schema and initial values.
+1. In DEPLOY_ZIP_HOME/setup folder, edit the file database.properties with properties matching your rdbms
+2. In DEPLOY_ZIP_HOME/setup/lib add your jdbc driver if needed (only for Microsoft SQL Server or Oracle)
```

Comment:
No, for other RDBMS as well, as you don't use the setup tool, and the drivers are not included in the war file